scripts/size-generate.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+rm -fr ./temp/size &&
+  pnpm run build:editor &&
+  pnpm run size:data &&
+  pnpm -C ./packages/playground run build:size &&
+  pnpm run --silent size:report >./size-report.md &&
+  cat ./size-report.md
```

Comment:
```suggestion
set -eu

rm -fr ./temp/size
pnpm run build:editor 
pnpm run size:data 
pnpm -C ./packages/playground run build:size 
pnpm run --silent size:report >./size-report.md 
cat ./size-report.md
```

The `-e` option tells the shell to exit immediately if any command within the script exits with a non-zero status.

`-u` treats unset variables and parameters as an error when performing parameter expansion.

I also believe that this can help us write more reliable shell.

https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html